CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_hubspot_source v0.14.2
+[PR #126](https://github.com/fivetran/dbt_hubspot_source/pull/126) includes the following updates:
+## Bugfix
+- Added field `_fivetran_end` to macro `get_ticket_property_history_columns()` to ensure the column available for use downstream.
+
+## Documentation
+- Update documentation to include `_fivetran_end` under model `stg_hubspot__ticket_property_history`.
```

Comment:
Would this _technically_ fall under a breaking change since this will change the underlying schema of the staging and downstream models?

Reply:
I know it _should_ be included within the below code. However, on the chance this wasn't included previously it would result in a breaking change. 

https://github.com/fivetran/dbt_hubspot_source/blob/81e916b8df5398c8f32d120d97da08ac4606bfa7/models/stg_hubspot__ticket_property_history.sql#L11-L14

Reply:
Yeah that is safer. And not much of a lift since the transform is breaking as well. Updated!